http-transport-binding.md

```diff
@@ -443,7 +446,18 @@ Content-Length: nnnn
 
 ```
 
-## 4. References
+## 4. HTTP Responses
+### 4.1. Size Limits
+When CloudEvents are forwarded through one or more generic intermediaries,
+a minimum payload size SHOULD be considered, in accordance to the
+[CloudEvents spec][ce-size].
+
+In the event of the payload being larger than what can be forwarded or
+accepted, the HTTP status code 413 MUST be sent back to the source.
```

Comment:
Overall I think the direction looks good. I would suggest some wording changes though. I wouldn't talk about just intermediaries (like you do in the 1st para) because then in the 2nd it seems inconsistent to switch to talk about generic CE receivers. Also, I wouldn't talk about sending thing back to the source because it may not be the source that initiated the http connection.

How about:
```
When CloudEvents are transported via HTTP, a minimm payload
size SHOULD be considered in accordance with the [CloudEvents spec][ce-size].
In the case of the payload of the incoming HTTP request being larger than what
can be accepted, the HTTP status code of `413 Payload Too Large` MUST be
sent in response. See [RFC7231][rfc2731-section-6-5-11] for more information
on this response code.
```

Reply:
I like this, let me update it.